### ISSUE TITLE
telegraf:alpine Docker container should pull in non-busybox ping (iputils)

### ISSUE THREAD

**Issue by Nemykal** [influxdata/influxdata-docker#60]
### Relevant telegraf.conf:
```
[[inputs.ping]]
 urls = [
   "8.8.8.8"
]
count = 1
```

### System info:

All versions. Using the telegraf:alpine container

### Steps to reproduce:

1. Run container that uses the telegraf ping plugin with a non-gnu 'ping', such as busybox's implementation

### Expected behavior:
Ping plugin works

### Actual behavior:
Telegraf errors with: 

```
/bin/ping: unrecognized option: i
BusyBox v1.25.1 (2016-10-26 16:15:20 GMT) multi-call binary.

Usage: ping [OPTIONS] HOST

Send ICMP ECHO_REQUEST packets to network hosts

        -4,-6           Force IP or IPv6 name resolution
        -c CNT          Send only CNT pings
        -s SIZE         Send SIZE data bytes in packets (default:56)
        -t TTL          Set TTL
        -I IFACE/IP     Use interface or IP address as source
        -W SEC          Seconds to wait for the first response (default:10)
                        (after all -c CNT packets are sent)
        -w SEC          Seconds until ping exits (default:infinite)
                        (can exit earlier with -c CNT)
        -q              Quiet, only display output at start
                        and when finished
        -p              Pattern to use for payload, exit status 1
Fatal error processing ping output
```

### Additional info:

See above. 

### Proposal:

Add 'iputils' to the Dockerfile for telegraf:alpine

[[ originally posted at https://github.com/influxdata/telegraf/issues/2351 ]]

**Comment by jsternberg** [comment]
I've tried installing `iputils` in a test and then I tried running it. I'm getting this even with iputils installed.

```
$ docker run --rm telegraf:1.2-alpine ping -h
ping: unrecognized option: h
BusyBox v1.25.1 (2016-10-26 16:15:20 GMT) multi-call binary.

Usage: ping [OPTIONS] HOST

Send ICMP ECHO_REQUEST packets to network hosts

        -4,-6           Force IP or IPv6 name resolution
        -c CNT          Send only CNT pings
        -s SIZE         Send SIZE data bytes in packets (default:56)
        -t TTL          Set TTL
        -I IFACE/IP     Use interface or IP address as source
        -W SEC          Seconds to wait for the first response (default:10)
                        (after all -c CNT packets are sent)
        -w SEC          Seconds until ping exits (default:infinite)
                        (can exit earlier with -c CNT)
        -q              Quiet, only display output at start
                        and when finished
        -p              Pattern to use for payload
```

`telegraf:1.2-alpine` is what I have it named locally after building the image. It is not the image that's on Docker Hub for the above.

Are you sure that `iputils` installs the GNU ping program? It seems to just install the same BSD one.

**Comment by jsternberg** [comment]
@sparrc what ping program does the FreeBSD implementation of telegraf use?

**Comment by sparrc** [comment]
it uses BSD

**Comment by jsternberg** [comment]
Any idea what might be the problem here then? Which ping does busybox use and is it possible to get telegraf to support that? I don't think there's another ping program I can install.

**Comment by sparrc** [comment]
I guess the busybox ping doesn't support `-i`, is there a runtime.GOOS for busybox? if so we could do something here: https://github.com/influxdata/telegraf/blob/master/plugins/inputs/ping/ping.go#L144

**Comment by gamkiller77** [comment]
So been looking in to this as I really want the alpine Linux version to work. From what I read the -i is not supported but -c is witch may work. I am also going to look in to removing the busybox ping and replace it with iputils as that would fix the -i problem. As soon as I get back from lunch I will look in to this. If it works what the projects method of getting this update?  A PR or just post here the new dockerfile lines needed to make it work?

**Comment by gamkiller77** [comment]
I was able to fix this problem with iputils being installed.  There was a line in the dockfile that kept removing it.  I am going to submit a PR @jsternberg @sparrc @Nemykal

**Comment by gamkiller77** [comment]
added iputils so ping works #68 << PR

**Comment by gamkiller77** [comment]
submitted new PR for both 1.1 and 1.2 releases.

**Comment by jsternberg** [comment]
Fixed via #69.
